include/z64collision_check.h

```diff
@@ -272,6 +272,8 @@ typedef enum {
     /* 7 */ ELEMTYPE_UNK7
 } ElementType;
 
+#define SAC_ON (1 << 0)
```

Comment:
the searching issue is easily remedied by the "match whole word" button, so imo dont let that be the reason you change it. If it ruins consistency with AC_ON and related flags (it may not at all, I dont know this system very well)

Reply:
SAC is an unused system allows you to manually overwrite the collider stored at a specified index in an AT/AC/OT list with a new one, instead of pending that collider to the end of the list

Reply:
My point was in fact to ruin consistency with AC_ON and others :p, since it has very little to do with those. Or did fig mean _ON is a common occurrence for defines in the repo? That wasn't my impression

Reply:
I was being careful incase they were kinda related or parallel in some sense. but if they are not then its not an issue, apologies for not having the background knowledge there

Reply:
I discovered this SAC thing only when reviewing this too lol, it's unused, SAC is from debug strings apparently.